app/components/page-header/template.hbs

```diff
@@ -45,7 +45,7 @@
             {{#each item.submenu as |subitem|}}
               {{#if subitem.route}}
                 {{#if (or (not subitem.resource) (rbac-allows resource=subitem.resource scope=subitem.resourceScope))}}
-                  <li class="{{if (and subitem.disableIfClusterNotReady (not cluster.isReady)) "pointer-events-none"}}">
```

Comment:
Use the condition (like [here](https://github.com/rancher/ui/blob/34b99436df8ca4efbf63625ad4325102124aa48d/app/instance-initializers/nav.js#L245-L247)) to hide it instead of having an unclickable item.  (The nav tree already automatically recomputes when cluster.isReady changes)